File: api/v2010/authorized_connect_app/fetch-example-1/fetch-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var authorizedConnectApp = AuthorizedConnectAppResource.Fetch(
            pathConnectAppSid: "CN47260e643654388faabe8aaa18ea6756"
        );

        Console.WriteLine(authorizedConnectApp.ConnectAppCompanyName);
    }
}

File: api/v2010/authorized_connect_app/fetch-example-1/fetch-example-1.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$authorized_connect_app = $twilio->authorizedConnectApps("CN47260e643654388faabe8aaa18ea6756")
                                 ->fetch();

print($authorized_connect_app->connectAppCompanyName);